create/mysql.go

```diff
@@ -44,7 +44,7 @@ func (cmd *mySQLCmd) Run(ctx context.Context, client *api.Client) error {
 	}
 	cmd.SSHKeys = append(cmd.SSHKeys, sshkeys...)
 
-	fmt.Println("Creating new mysql. This can take up to 15 minutes.")
+	fmt.Println("Creating new mysql. This can take up to 25 minutes.")
```

Comment:
Here I am actually not sure. Shouldn't we refer to the set `WaitTimeout` field in the message as they might divert otherwise? Also the message could be like:

```
Creating new mysql. This might take some time (waiting up to <xx> minutes).
```